lib/docs/main.js

```diff
@@ -51,6 +51,7 @@ var zigAnalysis;
   const domHdrName = document.getElementById("hdrName");
   const domHelpModal = document.getElementById("helpModal");
   const domSearchPlaceholder = document.getElementById("searchPlaceholder");
+  const sourceFileUrlTemplate = "https://github.com/ziglang/zig/blob/master/lib/std/{{file}}#L{{line}}"
```

Comment:
This would break with any changes to the standard library and would need to include the exact commit to work. I'm not sure if this is the right place to put it.

Also, is it expected that this will be extended to work for project specific documentation?